packages/tenant-process/src/services/tenantService.ts

```diff
@@ -182,6 +187,28 @@ export function tenantServiceBuilder(
         );
       }
     },
+    async getCertifiedAttributes({
+      organizationId,
+      offset,
+      limit,
+    }: {
+      organizationId: TenantId;
+      offset: number;
+      limit: number;
+    }): Promise<ListResult<ApiCertifiedAttribute>> {
+      const tenant = await readModelService.getTenantById(organizationId);
+
+      assertTenantExists(organizationId, tenant);
+      assertTenantIsCertifier(tenant.data);
+
+      const certifierId = tenant.data.features[0].certifierId;
```

Comment:
tenants may have more than one feature in the future, if a tenant is a certifier, but it is its second feature, this part would break.

Let's use a future proof approach, for example in Scala code the expression
```
requesterTenant.features.collectFirstSome(_.certifier.map(_.certifierId))
```
takes the first feature which has an existing `certifierId`, returns `None` (in TS `undefined`) otherwise.

Can we implement something similar?

Reply:
Sure! 👍